src/comps/blogList.tsx

```diff
@@ -0,0 +1,20 @@
+function BlogList(props) {
+  const { blogsList } = props;
+  const { blogTitle } = props;
+
+  return (
+    <div className="blogList">
+      <h2> {blogTitle} </h2>
```

Comment:
i saw that this one is not even important or being used since you already put everything inside the loop so you can just deleted it